wasm/wazero/module.go

```diff
@@ -231,3 +284,17 @@ func AddHostFunctions(ctx context.Context, runtime wazero.Runtime, moduleName st
 	}
 	return build.Compile(ctx)
 }
+
+func addUnboundedModule(ctx context.Context, runtime wazero.Runtime, moduleName string, imports map[string]api.FunctionDefinition) (wazero.CompiledModule, error) {
+	build := runtime.NewHostModuleBuilder(moduleName)
+
+	for importName, f := range imports {
+		build.NewFunctionBuilder().
+			WithGoFunction(api.GoFunc(func(ctx context.Context, stack []uint64) {
+				panic(fmt.Errorf("you are trying to call %q from module %q for which Substreams provided a dummy"+
```

Comment:
is there not a better way to add logs to the module's output?
Does that bubble up through the gRPC connection properly?

Reply:
This does trickles down to the gRPC connection